DigitalLearningSolutions.Web/Scripts/common.ts

```diff
@@ -23,3 +23,17 @@ export function sendBrowserAgnosticEvent <T extends HTMLElement>(
 
   return event;
 }
+
+export function selectAll(selectorClass: string): void {
+  const allCheckboxes = document.querySelectorAll(selectorClass) as NodeListOf<HTMLInputElement>;
+  allCheckboxes.forEach((checkbox) => {
+    if (!checkbox.checked) checkbox.click();

```

Comment:
I didn't originally write this, but having tried changing to that, it produces a lint error no-param-reassign. I assume this is why it's been done like this.

Reply:
I've [looked up that error](https://eslint.org/docs/rules/no-param-reassign#options) and it's unnecessary imo (the base error against reassigning arguments it sensible, but I don't understand what the problem is with reassigning properties on those arguments).
I can see we temporarily disable it with a comment elsewhere in the codebase, but if this works already then it might not be worth changing.